metcdv3/balancer.go

```diff
@@ -81,9 +81,6 @@ func (e *etcdClusterState) NodeTaskCount() (map[string]int, error) {
 			}
 			// We want to only include those nodes which were initially included,
 			// as some nodes may be shutting down, etc, and should not be counted
```

Comment:
nice we don't have to recheck the filter here because we check if the node is in the state map.